# Test cases automatically generated by Pynguin (https://www.pynguin.eu).
# Please check them before you use them.
import py_backwards.utils.helpers as module_0
import py_backwards.exceptions as module_1


def test_case_0():
    str_0 = " 2w&{v\x0bl}8]"
    none_type_0 = module_0.debug(str_0)


def test_case_1():
    variables_generator_0 = module_0.VariablesGenerator()


def test_case_2():
    bool_0 = True
    callable_0 = module_0.eager(bool_0)
    callable_1 = module_0.eager(callable_0)
    callable_2 = module_0.eager(bool_0)


def test_case_3():
    bool_0 = False
    none_type_0 = module_0.debug(bool_0)
    module_0.get_source(none_type_0)


def test_case_4():
    str_0 = "~.7S?U}"
    none_type_0 = module_0.warn(str_0)


def test_case_5():
    str_0 = "^Uw&Pv\x0bl}8]"
    var_0 = module_1.TransformationError(str_0, str_0, str_0, str_0)
    var_1 = module_0.eager(var_0)
    var_1.__call__(str_0, str_0, str_0, start=str_0)
